app/migrations/Version20180130155207.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180130155207 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('CREATE TABLE sylius_url_redirects (id INT AUTO_INCREMENT NOT NULL, oldRoute VARCHAR(255) NOT NULL, newRoute VARCHAR(255) NOT NULL, enabled TINYINT(1) NOT NULL, UNIQUE INDEX one_enabled_route (oldRoute, enabled), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
```

Comment:
Index prevents multiple `oldRoute' entries with `enabled = 0`.

Reply:
Yes, that is indeed a problem. I have replaced it with a validator.